torch/_inductor/coordinate_descent_tuner.py

```diff
@@ -75,6 +75,11 @@ def get_rmax(self):
             # large enough. We should not pick this large RBLOCK anyway
             return 2**30
 
+    def get_warpsmax(self):
+        # Currently, CUDA has a maximum of 1024 threads, so 32 is the max
```

Comment:
nit: 'CUDA has a maximum of 1024 threads' -> 'CUDA has a maximum of 1024 threads per block'?